Tasks/VsTest/constants.ts

```diff
@@ -0,0 +1,15 @@
+export module errorCodes {
+    export const publishResults = 'publishResults';
+    export const invokeVsTest = 'invokeVsTest';
+    export const runTestsLocally = 'runTestsLocally';
+    export const invalidSettingsFile = 'invalidSettingsFile';
+    export const executeVstest = 'executeVstest';
```

Comment:
keep the actual string value in lowercase

Reply:
I installed. It didn't recommend i make it Capital.

---
In reply to: [136024318](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136024318) [](ancestors = 136024318)

Reply:
done

---
In reply to: [136024318](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136024318) [](ancestors = 136024318)